<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>630</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1024</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1054</width>
    <height>682</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Noto Sans</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Connection</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>1024</width>
     <height>630</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>1024</width>
     <height>630</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">* {
	color: #000;
	border: none;
}

#centralwidget {
	background-image: url(&quot;:/images/images/login.png&quot;);
	background-repeat: no repeat;
	color: #eff9fe;
}

#formBg {
	background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:0.715909, stop:0 rgba(0, 0, 0, 15), stop:0.375 rgba(0, 0, 0, 55), stop:0.835227 rgba(0, 0, 0, 98));
}

QLabel , #showPassword{
	color:#DED;
}

QLineEdit {
	background-color:rgba(0, 0, 0, 80);
	border-radius: 6px;
	border:none;
	border-bottom:2px solid rgba(105, 118, 132, 255);
	color:rgba(255, 255, 255, 230);
	padding:7px;
}

QPushButton {	
	background-color: rgba(80, 80, 100, 150);
	color:rgba(155, 168, 182, 220);
	border-radius: 6px;
} 
QPushButton:hover {	
	color:rgba(170, 192, 200, 250);
}
QPushButton:pressed{	
	padding-left:5px;
	padding-top:5px;
	color:rgba(115, 128, 142, 255);
}

#errLbl {
	color: #F7BFB4;
}</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item alignment="Qt::AlignRight">
     <widget class="QWidget" name="form" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>0</height>
       </size>
      </property>
      <widget class="QLabel" name="formBg">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>0</y>
         <width>400</width>
         <height>631</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>400</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>190</y>
         <width>151</width>
         <height>51</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans</family>
         <pointsize>18</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Connection</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="username">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>270</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Username</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="password">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>340</y>
         <width>211</width>
         <height>41</height>
        </rect>
       </property>
       <property name="placeholderText">
        <string>Password</string>
       </property>
      </widget>
      <widget class="QPushButton" name="loginBtn">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>420</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Connect</string>
       </property>
      </widget>
      <widget class="QLabel" name="errLbl">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>390</y>
         <width>381</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Noto Sans</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="password_eye">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>340</y>
         <width>31</width>
         <height>41</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../resource.qrc">
         <normaloff>:/icons/icons/others/hide_pw.png</normaloff>:/icons/icons/others/hide_pw.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resource.qrc"/>
 </resources>
 <connections/>
</ui>
